sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -236,4 +246,97 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
       .addOrderBy("assessment.submittedDate", "DESC")
       .getMany();
   }
+
+  /**
+   * Creates a manual assessment for the application.
+   * @param applicationId application id.
+   * @param userId user id who triggered the manual reassessment.
+   */
+  manualReassessment(applicationId: number, note: string, userId: number) {
+    return this.dataSource.transaction(async (transactionalEntityManager) => {
+      const currentUser = { id: userId } as User;
+      const now = new Date();
+      const applicationRepo =
+        transactionalEntityManager.getRepository(Application);
+      const application = await applicationRepo.findOne({
+        select: {
+          id: true,
+          currentAssessment: true as unknown,
```

Comment:
Please select the necessary columns, `true as unknown` should be used only for JSONB columns.